READMEs/install.md

```diff
@@ -44,12 +44,20 @@ pip3 install --pre ocean-lib
 #### Create your network configuration file
```

Comment:
This is not for the creation of the network configuration file, cause that comes out of the box with the brownie installation.
My take on this is that we should move this section way below as an information addition. Cause ocean.py should work out of the box with the default config. The only case for people to edit that file is to add the energy web chain or to configure the gas prices..

Reply:
@trentmc should we rename the title and move the section as @AnaLoznianu suggested?